resources/zib-ContactInformation-TelephoneNumbers.xml

```diff
@@ -0,0 +1,166 @@
+      <comment value="Implementation of the &quot;TelephoneNumbers&quot; concept of zib ContactInformation.&#xD;&#xA;&#xD;&#xA;Please note that, although the zib concepts TelecomType and NumberType are functionally equivalent to the FHIR concepts `ContactPoint.system` and `ContactPoint.use` respectively, a mismatch occurs on the concept of mobile phone numbers; the zib uses the first concept and FHIR the second. For this reason, the ValueSet on `ContactPoint.system` is too narrow and the zib code will need to be communicated using the [codeSpecification extension](http://nictiz.nl/fhir/StructureDefinition/ext-codeSpecification).&#xD;&#xA;&#xD;&#xA;For the entire zib, the complementary profile [zib-ContactInformation-E-mailAddresses](http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-E-mailAddresses) should be used. Both profiles can be added in a slice with the discriminator set to `profile`/`$this`." />
+      <alias value="Telefoonnummers" />
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.2" />
+        <comment value="TelephoneNumbers" />
+      </mapping>
+    </element>
+    <element id="ContactPoint.extension">
+      <path value="ContactPoint.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="ContactPoint.extension:comment">
+      <path value="ContactPoint.extension" />
+      <sliceName value="comment" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+      </type>
+    </element>
+    <element id="ContactPoint.extension:comment.url">
+      <path value="ContactPoint.extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+    </element>
+    <element id="ContactPoint.extension:comment.value[x]">
+      <path value="ContactPoint.extension.value[x]" />
+      <short value="Comment" />
+      <definition value="Explanation about the telephone number and how to use it. It can be indicated, for example, that it is a department number (for healthcare professionnels) or that availability by phone is only possible during a specified part of the day." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.9" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="ContactPoint.system">
+      <path value="ContactPoint.system" />
+      <short value="TelecomType" />
+      <definition value="The telecom or device type that the telephone number is connected to." />
+      <alias value="TelecomType" />
+      <min value="1" />
+      <binding>

```

Comment:
Ik twijfel of we in een comment moeten neerzetten dat we de FHIR Required valueset hebben ingeperkt, niet dat we eigen waardes hebben neergezet. Misschien overbodig, wat denk jij?

Reply:
Ik twijfel of het toegevoegde waarde heeft. Het is extra guidance, maar ook weer extra tekst om te lezen terwijl je het ook kan zien aan de waardelijst. En als implementer ben je denk ik vooral geïnteresseerd in de vraag hoe je van zib naar FHIR komt, niet dat de lijst op die slice is ingeperkt -- dat is een gevolg.

Reply:
Het is inderdaad van deze relatief korte waardelijst (makkelijk) af te leiden dat het om een inperking gaat. Daarnaast staat het ook al in de ValueSet zelf. Dus wel afdoende.